### ISSUE TITLE
[Suggestion]: Phase Percentage NBT

### ISSUE THREAD

**Issue by RepositoryLover** [nonamecrackers2/crackers-wither-storm-mod#1864]
### Description of feature request

Update the phase NBT tag like this: /witherstormmod phase set @w 3 60%

**Comment by TaintedBlock** [comment]
A way to set and get the percentage of the storm progress? That would be helpful actually. Although, the first part can be fixed with the consumed entities command

**Comment by RepositoryLover** [comment]
> A way to set and get the percentage of the storm progress? That would be helpful actually. Although, the first part can be fixed with the consumed entities command

Yes. A way to set and get the percentage of the Wither Storm progress.
